pom.xml

```diff
@@ -78,7 +78,7 @@
 
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <cdap.version>4.2.0-SNAPSHOT</cdap.version>
+    <cdap.version>4.3.0-SNAPSHOT</cdap.version>
```

Comment:
also bump this part to 4.3.0-SNAPSHOT:
https://github.com/hydrator/wrangler/pull/126/files#diff-9489ccc8dfd9ca9f33b5347d0aa74244R18